aws-event-bridge/src/main/scala/akka/stream/alpakka/aws/eventbridge/EventBridgePublishSettings.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.aws.eventbridge
+
+final class EventBridgePublishSettings private (val concurrency: Int) {
```

Comment:
Can you add some docs here?  Maybe you can add some external references to upstream AWS Eventbridge docs (java or plain docs) about `concurrency`.